.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - run:
           name: Run Core tests
-          command: env PYTEST_ADDOPTS="--exitfirst" make test
+          command: make test pytest_args="--exitfirst"
```

Comment:
I would love to have different steps for Core, Country Template and Extension Template visible in CI 🙂